sass/_global.scss

```diff
@@ -22,3 +22,26 @@ body {
 .region-sidebar-first {
   padding: 1.667em 0;
 }
+
```

Comment:
this is a bootstrap override, right?
I think we should have a specific file for these overrides instead of polluting our global.scss, drupal_hacks, beside the fact that we would love to get rid of that, could be a better place for this, for now.